### PR TITLE
[Documentation] Fix broken links

### PR DESCRIPTION
**Pull request by colinepiloquet**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

No

**What kind of change does this PR introduce?**

Docs update

**What is the current behavior?**

Some links are broken in the documentation, they were not yet updated after the creation of the repositories.

**What is the new behavior (if this is a feature change)?**
Fixes the dead links.

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [X] No

### CODE REVIEW

File: docs/simulation/loadflow/index.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ active power (zero MW if slack distribution is disabled).
...
-- [PowSyBl OpenLoadFlow](TODO)
-- [Dynaflow](TODO)
+- [PowSyBl OpenLoadFlow](inv:powsyblopenloadflow:*:*#index)
+- [Dynaflow](inv:powsybldynawo:*:*#index)
 
 ## Going further
 - [Run a power flow through an iTools command](../../user/itools/loadflow.md): Learn how to perform a power flow calculation from the command line
-- [Load flow tutorial](../../developer/tutorials/loadflow.md)
\ No newline at end of file
+- [Load flow tutorial](TODO)
```
Comment (colinepiloquet) [inline docs/simulation/loadflow/index.md:49]: This will need to be added later, tutorials is not available yet.

**Comment by colinepiloquet** [comment]
There are still a lot of broken links for the user part of the documentation, but I think this section might change in the future. Some links are also referring to non existing pages.

**Comment by colinepiloquet** [comment]
Many links are empty, I am not sure what I should do.

**Review by So-Fras — APPROVED** [review]

**Comment by So-Fras** [comment]
Let's start with fixing **some** links. It is a huge work to fix all links in one PR.

**Pull request merged**